client/hotrod-client/src/test/java/org/infinispan/client/hotrod/near/AssertsNearCache.java

```diff
@@ -33,25 +34,30 @@
    final BlockingQueue<MockEvent> events;
    final RemoteCacheManager manager;
    final NearCacheMode nearCacheMode;
+   final AtomicReference<NearCacheService<K, V>> nearCacheServiceByRef;
 
-   private AssertsNearCache(RemoteCacheManager manager, Cache<byte[], ?> server, BlockingQueue<MockEvent> events) {
+   private AssertsNearCache(RemoteCacheManager manager, Cache<byte[], ?> server, BlockingQueue<MockEvent> events, AtomicReference<NearCacheService<K, V>> nearCacheServiceByRef) {
       this.manager = manager;
       this.remote = manager.getCache();
       this.server = server;
       this.events = events;
       this.nearCacheMode = manager.getConfiguration().nearCache().mode();
+      this.nearCacheServiceByRef = nearCacheServiceByRef;
    }
 
    static <K, V> AssertsNearCache<K, V> create(Cache<byte[], ?> server, ConfigurationBuilder builder) {
       final BlockingQueue<MockEvent> events = new ArrayBlockingQueue<>(128);
+      AtomicReference<NearCacheService<K, V>> nearCacheServiceAtomicReference = new AtomicReference<>();
```

Comment:
`nearCacheService` should be enough IMO, at most `nearCacheServiceRef`

Reply:
IntelliJ just recommended - I can shorten

Reply:
Changed to `nearCacheServiceRef` to not clash with the other `nearCacheService` in the overridden method.